<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificarMaterial</class>
 <widget class="QWidget" name="ModificarMaterial">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Material</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelNombre">
       <property name="text">
        <string>Nombre:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditNombre"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelCantidad">
       <property name="text">
        <string>Cantidad:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinBoxCantidad">
       <property name="minimum">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelEstado">
       <property name="text">
        <string>Estado:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxEstado">
       <item>
        <property name="text">
         <string>Bueno</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Dañado</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>En reparación</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Regular</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Malo</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelUbicacion">
       <property name="text">
        <string>Ubicación:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="lineEditUbicacion"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelFechaIngreso">
       <property name="text">
        <string>Fecha Ingreso:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="labelFechaValor">
       <property name="text">
        <string>--/--/----</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="btnGuardar">
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
